### ISSUE TITLE
GitHub workflow linting (?) error popups are visually broken

### ISSUE THREAD

**Issue by exalted** [zed-industries/community#1309]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

See the screenshot

### Environment

Zed: v0.78.1 (stable)
OS: macOS 13.2.1
Memory: 64 GiB
Architecture: aarch64

### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.
If you only need the most recent lines, you can run the `zed: open log` command palette action to see the last 1000.

_No response_

**Comment by iamnbutler** [comment]
Woah freaky.

**Comment by JosephTLyons** [comment]
Yeah, this was fixed in v0.79.1.  Things are a bit hectic with issue - this was just missed when closing things out.
